tests/infra/remote.py

```diff
@@ -231,6 +236,21 @@ def start(self):
         cmd = self._cmd()
         LOG.info("[{}] {}".format(self.hostname, cmd))
         stdin, stdout, stderr = self.client.exec_command(cmd, get_pty=True)
+        _, stdout_, _ = self.proc_client.exec_command(f'ps -ef | pgrep -f "{cmd}"')
+        self.pid = stdout_.readline()
+
+    def suspend(self):
+        _, stdout, _ = self.proc_client.exec_command(f"kill -STOP {self.pid}")
+        if stdout.channel.recv_exit_status() == 0:
+            LOG.info(f"Node {self.name} suspended...")
+        else:
+            raise RuntimeError(f"Node {self.name} could not be suspended")
+
+    def resume(self):
+        _, stdout, _ = self.proc_client.exec_command(f"kill -CONT {self.pid}")
+        if stdout.channel.recv_exit_status() != 0:
+            raise RuntimeError(f"Could not resume node {self.name} from suspension!")
+        LOG.info(f"Node {self.name} resuming from suspension...")
```

Comment:
Minor: Should this say `"Node {self.name} resumed from suspension"` since by this point, we know the node has resumed?